README.md

```diff
@@ -85,77 +95,123 @@ docker-compose up -d elasticsearch
 
 ### 4. Populate test data
 
-Vendure uses [TypeORM](http://typeorm.io), and officially supports **MySQL**, **MariaDB**, **PostgreSQL** and **SQLite**.
+Vendure uses [TypeORM](http://typeorm.io) and officially supports **MySQL**, **MariaDB**, **PostgreSQL**, and **SQLite**.
 
 The first step is to populate the dev server with some test data:
 
 ```bash
 cd packages/dev-server
-
-[DB=mysql|postgres|sqlite] npm run populate
+npm run populate
 ```
 
-If you do not specify the `DB` variable, it will default to "mysql". If you specifically want to develop against Postgres,
-you need to run the `postgres_16` container and then run `DB=postgres npm run populate`.
+By default, if you do not specify the `DB` environment variable, it will use **MySQL/MariaDB**.
 
-### 5. Run the dev server
+If you want to develop against **PostgreSQL**:
 
+1. Run the `postgres_16` Docker container.
+```bash
+docker-compose up -d postgres_16
 ```
+2. Create a `.env` file in the root of your project (or in the package you're working on) and declare the variable inside it:
```

Comment:
"or in the package you're working on" - that won't work though - the .env file would need to be in the `/packages/dev-server` or root dir as far as I know. You can test this to see if I am right but I think the call to `import 'dotenv/config';` in the `dev-config.ts` file will look for an env file in the current dir and then up the tree.

Reply:
But no matter what package you work on, you will _always_ be running the `/packages/dev-server` package to actually test it. There are no separate dev servers inside the individual package dirs.